README.md

```diff
@@ -67,6 +67,9 @@ Each version of the client is tagged and the version is updated accordingly.
 
 To see the list of past versions, run `git tag`.
 
+To release a new version, first ensure that `[Version](./mongodbatlas/mongodbatlas.go)' is updated 
```

Comment:
should 
>`[Version](./mongodbatlas/mongodbatlas.go)'
be
`[Version](./mongodbatlas/mongodbatlas.go)` 
also does the linking works like this? sometimes for file you need to add blob in the path

Reply:
nice catch on the backticks!
Yup, linking like that (against a file or dir, relative to the current file) works due to GitHub Markdown syntax; e.g.: https://github.com/mongodb/go-client-mongodb-atlas/blob/ed14165966a89e84a67e79ad7aeabedbab2734b9/README.md